<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="HC595PIN1_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>65</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>65</y>
      <width>36</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BYTE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN1_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>65</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN2_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>88</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>88</y>
      <width>25</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>R/C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN2_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>88</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN4_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>134</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_4</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN4_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>134</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_4</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN3_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>111</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>111</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CS_7805</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN3_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>111</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>134</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CS_5615</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>249</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN7_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>202</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_7</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN8_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>225</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_8</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN6_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>179</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_6</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>203</y>
      <width>36</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RESV</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN7_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>202</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_7</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN5_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>157</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_5</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN6_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>179</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_6</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN8_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>225</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_8</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>157</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SNCLK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN5_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>157</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_5</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>180</y>
      <width>36</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RESV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>387</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD7</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN14_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>362</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_14</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>364</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD6</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN0_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>42</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CLRbuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN10_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>271</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_10</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>318</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>272</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>42</y>
      <width>28</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CLR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN11_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>293</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_11</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN13_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>339</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_13</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN0_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>42</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CLRbuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>295</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>341</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HC595PIN12_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>316</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_12</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN9_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>248</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_9</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN11_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>293</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_11</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN10_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>271</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_10</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN9_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>248</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_9</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN14_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>362</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_14</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN12_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>316</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_12</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN13_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>339</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_13</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN15_0">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>385</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_15</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="HC595PIN15_1">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>385</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_15</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>226</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PTD0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Reset595">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset595</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Get595">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Get595</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>HC595PIN1_0</zorder>
   <zorder>label</zorder>
   <zorder>HC595PIN1_1</zorder>
   <zorder>HC595PIN2_1</zorder>
   <zorder>label_2</zorder>
   <zorder>HC595PIN2_0</zorder>
   <zorder>HC595PIN4_0</zorder>
   <zorder>HC595PIN4_1</zorder>
   <zorder>HC595PIN3_1</zorder>
   <zorder>label_3</zorder>
   <zorder>HC595PIN3_0</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>HC595PIN7_0</zorder>
   <zorder>HC595PIN6_1</zorder>
   <zorder>label_6</zorder>
   <zorder>HC595PIN7_1</zorder>
   <zorder>HC595PIN5_1</zorder>
   <zorder>HC595PIN6_0</zorder>
   <zorder>label_7</zorder>
   <zorder>HC595PIN5_0</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>HC595PIN14_0</zorder>
   <zorder>label_10</zorder>
   <zorder>HC595PIN8_0</zorder>
   <zorder>HC595PIN0_0</zorder>
   <zorder>HC595PIN10_0</zorder>
   <zorder>HC595PIN8_1</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>HC595PIN11_0</zorder>
   <zorder>HC595PIN13_1</zorder>
   <zorder>HC595PIN0_1</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>HC595PIN12_1</zorder>
   <zorder>HC595PIN9_0</zorder>
   <zorder>HC595PIN11_1</zorder>
   <zorder>HC595PIN10_1</zorder>
   <zorder>HC595PIN9_1</zorder>
   <zorder>HC595PIN14_1</zorder>
   <zorder>HC595PIN12_0</zorder>
   <zorder>HC595PIN13_0</zorder>
   <zorder>HC595PIN15_0</zorder>
   <zorder>HC595PIN15_1</zorder>
   <zorder>label_19</zorder>
   <zorder>Reset595</zorder>
   <zorder>Get595</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_7"/>
  <buttongroup name="CLRbuttonGroup"/>
  <buttongroup name="buttonGroup_14"/>
  <buttongroup name="buttonGroup_10"/>
  <buttongroup name="buttonGroup_11"/>
  <buttongroup name="buttonGroup_13"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_8"/>
  <buttongroup name="buttonGroup_15"/>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_12"/>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup_9"/>
 </buttongroups>
</ui>
